frontend/src/containers/UserApplicationAdminView/index.js

```diff
@@ -15,34 +15,28 @@ import SmallDescriptionWrapper from "./SmallDescriptionWrapper";
 import Header from "./Header";
 import NumApplications from "./NumApplications";
 
-class UserApplicationAdminView extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      groupApplications: [],
-    };
-  }
-
-  componentDidMount() {
-    const {
-      user,
-      text,
-      group_applications,
-      created_at,
-      updated_at,
-      applied_within_deadline,
-      phone_number,
-      pk,
-    } = this.props;
+const UserApplicationAdminView = ({
+  user,
+  text,
+  group_applications,
+  created_at,
+  updated_at,
+  applied_within_deadline,
+  phone_number,
+  pk,
+  generateCSVData,
+}) => {
+  const [groupApplications, setGroupApplications] = useState([]);
 
+  useEffect(() => {
     group_applications.sort(function (a, b) {
       if (a.group.name < b.group.name) return -1;
       if (a.group.name > b.group.name) return 1;
       return 0;
     });
```

Comment:
Or use a `useMemo` like `const sorted = useMemo(() => [...group_applications].sort(...), [group_applications])` and get rid of the `useEffect`

Reply:
Or do you think that would not work because of shallow equality check @LudvigHz

Reply:
Just checked this, and `useMemo` compares on object reference. So essentially it will do the same as the useEffect one. I'm pretty sure both works though, since I think the reference will change on updates (unless we don something funky further up the tree)